dropwizard-jersey/src/test/java/io/dropwizard/jersey/errors/ErrorEntityWriterTest.java

```diff
@@ -0,0 +1,71 @@
+package io.dropwizard.jersey.errors;
+
+import com.codahale.metrics.MetricRegistry;
+import io.dropwizard.jersey.AbstractJerseyTest;
+import io.dropwizard.jersey.DropwizardResourceConfig;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.glassfish.jersey.servlet.ServletProperties;
+import org.glassfish.jersey.test.DeploymentContext;
+import org.glassfish.jersey.test.ServletDeploymentContext;
+import org.glassfish.jersey.test.grizzly.GrizzlyWebTestContainerFactory;
+import org.glassfish.jersey.test.spi.TestContainerException;
+import org.glassfish.jersey.test.spi.TestContainerFactory;
+import org.junit.Test;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.failBecauseExceptionWasNotThrown;
+
+public class ErrorEntityWriterTest extends AbstractJerseyTest {
+
+    public static class ErrorEntityWriterTestResourceConfig extends DropwizardResourceConfig {
+        public ErrorEntityWriterTestResourceConfig() {
+            super(true, new MetricRegistry());
+
+            register(DefaultLoggingExceptionMapper.class);
+            register(DefaultJacksonMessageBodyProvider.class);
+            register(ExceptionResource.class);
+            register(new ErrorEntityWriter<ErrorMessage, String>(MediaType.TEXT_HTML_TYPE, String.class) {
+                @Override
+                protected String getRepresentation(ErrorMessage entity) {
+                    return "Uh oh: " + entity.getMessage();
```

Comment:
Fair point - that is a bit misleading for anyone reading the test. I'll add some tags to make clear that it's HTML.